app/main.py

```diff
@@ -1,3 +1,94 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
```

Comment:
The type hint int for km is a bit restrictive. Since division can result in a float value being stored in self.km, it's good practice to allow floats here as well. Consider using km: int | float.